content/rs/administering/access-control/password-rotation.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Database Password Rotation
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+To maintain secure access to your Redis Enterprise Software (RS) cluster and databases,
+most organizational security policies require that you rotate user passwords on a regular basis.
```

Comment:
Maybe:

Most organizational security policies require periodic password rotation. Redis Enterprise Software allows you to implement these policies using its API. Specifically, you can add a new password for a database user without immediately invalidating the old one (which might cause authentication errors in production).